dap_prinz_green_jobs/analysis/ojo_analysis/occupation_similarity.py

```diff
@@ -0,0 +1,258 @@
+"""
+Get counts and proportions of skills in each occupation.
+Use these to create occupation skill similarity measures.
+"""
+
+import dap_prinz_green_jobs.analysis.ojo_analysis.process_ojo_green_measures as pg
+from dap_prinz_green_jobs.getters.data_getters import save_to_s3, load_s3_data
+from dap_prinz_green_jobs import BUCKET_NAME, PROJECT_DIR, logger, analysis_config
+from dap_prinz_green_jobs.getters.ojo_getters import (
+    get_mixed_ojo_location_sample,
+    get_mixed_ojo_salaries_sample,
```

Comment:
i don't think you are using these guys in this script so don't need to import

Reply:
VScode has nice visuals to see if you are actually using all your imports so easy to delete

Reply:
thanks! I really do need to use VS code!